assert/assertions.go

```diff
@@ -923,6 +932,8 @@ func Contains(t TestingT, s, contains interface{}, msgAndArgs ...interface{}) bo
 //	assert.NotContains(t, "Hello World", "Earth")
 //	assert.NotContains(t, ["Hello", "World"], "Earth")
 //	assert.NotContains(t, {"Hello": "World"}, "Earth")
+//	assert.NotContains(t, []byte("Hello World"), []byte("Earth"))
+//	assert.NotContains(t, [[]byte("Hello"), []byte("World")], []byte("Earth"))
```

Comment:
This line is not valid Go.

Also: any case presented in the doc must be strictly checked in the test suite.